lib/hexpm_web/views/dashboard/audit_log_view.ex

```diff
@@ -0,0 +1,100 @@
+defmodule HexpmWeb.Dashboard.AuditLogView do
+  use HexpmWeb, :view
+
+  alias HexpmWeb.DashboardView
+
+  @doc """
+  Translate an audit_log to user readable descriptions
+  """
+  def humanize_audit_log_info(log = %{action: "docs.publish"}) do
+    "Publish documentation for #{log.params["package"]["name"]} (#{
+      log.params["release"]["version"]
+    })"
+  end
+
+  def humanize_audit_log_info(log = %{action: "docs.revert"}) do
+    "Revert documentation for #{log.params["package"]["name"]} (#{
+      log.params["release"]["version"]
+    })"
+  end
+
+  def humanize_audit_log_info(log = %{action: "key.generate"}) do
+    "Generate key #{log.params["name"]}"
+  end
+
+  def humanize_audit_log_info(log = %{action: "key.remove"}) do
+    "Remove key #{log.params["name"]}"
+  end
+
+  def humanize_audit_log_info(log = %{action: "owner.add"}) do
+    "Add #{log.params["user"]["username"]} as a new owner of package #{
+      log.params["package"]["name"]
+    }"
+  end
+
+  def humanize_audit_log_info(log = %{action: "owner.transfer"}) do
+    "Transfer package #{log.params["package"]["name"]} to #{log.params["user"]["username"]}"
+  end
+
+  def humanize_audit_log_info(log = %{action: "owner.remove"}) do
+    "Remove #{log.params["user"]["username"]} from owners of package #{
+      log.params["package"]["name"]
+    }"
+  end
+
+  def humanize_audit_log_info(log = %{action: "release.publish"}) do
+    "Publish package #{log.params["package"]["name"]} version #{log.params["release"]["version"]}"
```

Comment:
Sometimes we have `package version 1.0.0` and sometimes `package (1.0.0)`, we should be consistent.

Reply:
updated in https://github.com/hexpm/hexpm/pull/886/commits/8a4ef6e9feccec8be98a0788fc7ebf0fb0dce3d3